x/consensus/keeper/filters/is_oldest_per_sender_filter.go

```diff
@@ -0,0 +1,27 @@
+package filters
+
+import evmtypes "github.com/palomachain/paloma/x/evm/types"
+
+// Biased: Filter expects messages to be ordered by message ID (default)
+// Filter requires injection of a look-up table to keep track of senders
+// Filter out multiple messages from same sender
+// This is done to avoid possible relays of more messages than
+// user is able to pay for.
+func IsOldestMsgPerSender(lut map[string]struct{}, msg *evmtypes.Message) bool {
```

Comment:
Since we only care about SLC messages here, should we rename this to something more specific? Perhaps `IsOldestSLCPerSender` instead?

Reply:
Hmm, at first I agreed. Then I realized `IsOldestSLCPerSender` is also kind of misleading, as this gets called for all messages, and will return for all other message types that might not be an SLC or have a concept of `sender`. 

So it would be called something like `IsOldestSLCOrNonSLCMessage`, which is starting to get silly. So I suppose while not 100% accurate, I'd leave it as is unless you feel strongly otherwise.

Reply:
Yeah, that makes sense. Let's keep it simple. :+1: